Sources/MongoSwiftSync/Exports.swift

```diff
@@ -1,4 +1,4 @@
-// Generated using Sourcery 0.16.1 — https://github.com/krzysztofzablocki/Sourcery
+// Generated using Sourcery 0.17.0 — https://github.com/krzysztofzablocki/Sourcery
```

Comment:
I think this needs to be 0.16.1 to not break travis. We should update this in the future to not include the sourcery version.

Reply:
is it possible to make sourcery exclude that line?

Reply:
Appears not unfortunately: https://github.com/krzysztofzablocki/Sourcery/issues/399